catroid/src/catroid/java/org/catrobat/catroid/common/FlavoredConstants.java

```diff
@@ -37,11 +37,17 @@
 
 	public static final String POCKET_CODE_EXTERNAL_STORAGE_FOLDER_NAME = "Pocket Code";
 
+	public static final String POCKET_CODE_EXTERNAL_EXPORT_STORAGE_FOLDER_NAME = "CATROBAT";
+
+	public static final String POCKET_CODE_ZIP_EXTENSION = ".CATROBAT";
+
 	public static final File DEFAULT_ROOT_DIRECTORY = CatroidApplication.getAppContext().getFilesDir();
 
 	public static final File EXTERNAL_STORAGE_ROOT_DIRECTORY = new File(
 			Environment.getExternalStorageDirectory().getAbsolutePath(), POCKET_CODE_EXTERNAL_STORAGE_FOLDER_NAME);
 
+	public static final File EXTERNAL_STORAGE_ROOT_EXPORT_DIRECTORY = new File(
+			Environment.getExternalStorageDirectory().getAbsolutePath(), POCKET_CODE_EXTERNAL_EXPORT_STORAGE_FOLDER_NAME);
```

Comment:
using `Environment.getExternalStorageDirectory()` is enough, you don't need the `.getAbsolutePath()` part.